react/Viewer/PdfJsViewer.jsx

```diff
@@ -140,15 +140,18 @@ export class PdfJsViewer extends Component {
                 pageNumber={page + 1}
                 width={pageWidth}
                 renderAnnotations={false}
-                className={cx('u-mv-1', styles['viewer-pdfviewer-page'])}
+                className={cx(
+                  'u-mv-1 page-react-pdf',
+                  styles['viewer-pdfviewer-page']
+                )}
               />
             ))
           ) : (
             <Page
               pageNumber={currentPage}
               width={pageWidth}
               renderAnnotations={false}
-              className={styles['viewer-pdfviewer-page']}
+              className={cx('page-react-pdf', styles['viewer-pdfviewer-page'])}
```

Comment:
Added class to make test work, I'm at a loss here why the test doesn't work

Reply:
On devrait plutôt nommer la classe en fonction de l'utilisation, genre test-page-react-pdf ou quelque chose du genre. 

Si c'est pour le `component.find('.page-react-pdf')` plus bas, on devrait plutôt utiliser le data-testid que la class css

Reply:
- [ ] Utiliser data attribute

Reply:
rolled back to       `const pages = component.find('Page')`